core/src/tangram.cpp

```diff
@@ -94,6 +95,7 @@ Map::~Map() {
     // The unique_ptr to Impl will be automatically destroyed when Map is destroyed.
     TextDisplay::Instance().deinit();
     Primitives::deinit();
+    QuadIndices::reset();
```

Comment:
Maybe not the right place to put this.
(Maybe RenderState destructor is the right place for this reset)
Or reintroduce `meshCounter` thingy which is being done on master.